.ci.yaml

```diff
@@ -1502,7 +1502,7 @@ targets:
 
   - name: Linux web_benchmarks_canvaskit
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    presubmit: true
```

Comment:
Do we want to flip these to presubmit permanently?

Reply:
Nope, I'm enabling them temporarily to check that benchmarks are still running and I didn't break something.